app/grandchallenge/reader_studies/templates/reader_studies/readerstudy_detail.html

```diff
@@ -83,12 +83,31 @@ <h2>Editors</h2>
         </p>
 
         <h2>Readers</h2>
-        <ul>
-            {% for user in object.readers_group.user_set.all %}
-                <li>{{ user|user_profile_link }}</li>
+        <ul class="list-group list-group-flush">
+            {% for reader in readers %}
+                <li class="list-group-item">
+                    <div class="container">
+                        <div class="row">
+                            <div class="col-md-4 mb-1">
+                                <div>{{ reader.obj|user_profile_link }}</div>
+                            </div>
+                            <div class="col-md-8">
+                                <div class="progress">
+                                    <div class="progress-bar" role="progressbar" style="width: {{ reader.progress.hangings }}%" aria-valuenow="{{ reader.progress.hangings }}" aria-valuemin="0" aria-valuemax="100">
+                                        Hangings: {{ reader.progress.hangings|floatformat }}%
+                                    </div>
+                                    <div class="progress-bar bg-info" role="progressbar" style="width: {{ reader.progress.diff }}%" aria-valuenow="{{ reader.progress.diff }}" aria-valuemin="0" aria-valuemax="100">
+                                        Total questions: {{ reader.progress.questions|floatformat }}%
+                                    </div>
```

Comment:
Showing both is a little messy, let's stick with just the `Hangings:` right now, and call that `Progress:` instead

Reply:
I've changed it in the template only. Do you want to remove the other measure completely or do you think it might be useful at some point?

Reply:
I think that's fine, it might be useful later and is low overhead so lets keep it.